src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,60 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+  public static final String REGEX_DELIMITER = "|";
+  public static final String SEPARATOR = ",|:";
+  public static final Pattern REGEX_SEPARATOR = Pattern.compile("//(.)\n(.*)");
+
+  public static int splitAndSum(String text) {
+    if (isBlank(text)) {
+      return 0;
+    }
+    return sum(toInts(split(text)));
+  }
+
+  private static String[] split(String text) {
+    Matcher separatorMatcher = REGEX_SEPARATOR.matcher(text);
+    String customDelimiter = SEPARATOR;
+    if (separatorMatcher.find()) {
+      customDelimiter = String.join(REGEX_DELIMITER, customDelimiter, separatorMatcher.group(1));
+      text = separatorMatcher.group(2);
+    }
+    return text.split(customDelimiter);
+  }
+
+  private static String parseDelimiter(Matcher m) {
+    return SEPARATOR + REGEX_DELIMITER + m.group(1);
+  }
+
+  private static boolean isBlank(String text) {
+    return text == null || text.isBlank();
+  }
+
+  private static int sum(int[] numbers) {
+    int result= 0;
+    for (int number : numbers) {
+      result += number;
+    }
+    return result;
```

Comment:
`numbers`가 `List`라는 가정하에 
이런 부분들은 `stream`을 사용하시면 훨씬 간결하게 표현하실 수 있을 것 같습니다 😄 

```suggestion
    return return numbers.stream().reduce(0, Integer::sum);
```

그리고 
`java` 에는 `List`라는 훌륭한 컬렉션을 제공하고 있습니다
배열일 경우 물론 세부적인 `element`를 제어할 수 있다는 장점이 있지만
항상 `indexing`에 신경써야 하고 (for문을 통한)
`ArrayIndexOutOfBoundsException`을 항상 걱정해야 합니다 😓

앞으로 이 부분도 좀 더 연습해보시는걸 추천 드립니다 😄